src/background/user-storage.ts

```diff
@@ -2,13 +2,18 @@ import {DEFAULT_SETTINGS, DEFAULT_THEME} from '../defaults';
 import {debounce} from '../utils/debounce';
 import {isURLMatched} from '../utils/url';
 import {UserSettings} from '../definitions';
-import {readSyncStorage, readLocalStorage, writeSyncStorage, writeLocalStorage} from './utils/extension-api';
+import {readSyncStorage, readLocalStorage, writeSyncStorage, writeLocalStorage, isWriting} from './utils/extension-api';
```

Comment:
Maybe I miss something, maybe it works the other way in transpiled code, but I'm sure that `isWriting` will always be a constant value. Use `someObject.isWriting` instead. Anyway no need in it, see below...